publish-test-reports.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+QA_USER="qaplanetv1"
+
+current_folder=$(echo $PWD | awk -F "/" '{print $NF}')
+if [ "$current_folder" != "gen3-qa" ]; then
+    echo "please run this script from the root folder of your Gen3 QA workspace. e.g., cd ~/workspace/gen3-qa"
+    exit 1
+fi
+
+# if a custom folder is not provided, expect to find all the CodeceptJS testsuite .xml files
+# inside the "gen3-qa/output" folder 
+if [ -z $1 ]; then test_results_folder="output"; else test_results_folder="$1"; fi
+
+which allure
+RC=$?
+if [ $RC -ne 0 ]; then
+    echo "The allure CLI is not installed. Please follow the instructions on https://github.com/allure-framework/allure2#download to proceed."
+    exit 1
+fi
+
+current_timestamp=$(date "+%Y%m%d_%H%M%S")
+gen3_qa_report_folder="gen3_qa_report_${current_timestamp}"
+
```

Comment:
maybe take a folder name on the command line, so it's easy to pick out the report for some particular run?
just add the date-time as a suffix?

Reply:
Ok, I've tweaked the script to take 2 arguments: The `location of the test results` (.xml files) and the `name of the report`.
Here's how we should run it:
```
./publish-test-reports.sh output dcf_release_test
/usr/local/bin/allure
Report successfully generated to dcf_release_test_20191212_125533
```